dotcom-rendering/src/web/components/ArticleHeadline.tsx

```diff
@@ -223,6 +223,39 @@ const zIndex = css`
 	z-index: 1;
 `;
 
+const ageWarningMargins = (format: ArticleFormat) =>
+	format.display === ArticleDisplay.Immersive
+		? css`
+				margin-left: 0px;
+				margin-bottom: 0px;
+
+				${from.tablet} {
+					margin-left: 10px;
+				}
+
+				${from.leftCol} {
+					margin-left: 20px;
+				}
+		  `
+		: css`
+				margin-top: 12px;
+				margin-left: -10px;
+				margin-bottom: 6px;
+
+				${from.tablet} {
+					margin-left: -20px;
+				}
+
+				${from.leftCol} {
+					margin-left: -10px;
+					margin-top: 0;
+				}
+		  `;
+
+const backgroundStyles = (palette: Palette) => css`
+	background-color: ${palette.background.headline};
+`;
```

Comment:
I’m wondering if in cases such as these, we want to make it more explicit which part of the palette we require, maybe like so:

```suggestion
const backgroundStyles = (background: Palette['background']) => css`
	background-color: ${background.headline};
`;
```

Reply:
Does this speak to our discussion around how much we want to lean into typescript vs. keeping things simple? I can see how this might seem like a nicer, more concise solution, and in isolation that is indeed the case, but perhaps the larger impact here is that we're incrementally adding more and more features of Typescript, slowly increasing the breadth of knowledge that we're expecting new developers to have have before they can easily read and understand this code?

I realise the instinct when reading this is to think, but this is easy! When you have Thing['property'] then this means the type being used in only that properties type, not the whole thing. But the issue is when you add up all these small, seemingly small, simple little things you end up with one large, complicated thing which overwhelms people who haven't had the chance to slowly learn them step by step

Reply:
I was thinking more of the way we structure `palette`. As an example, `common-rendering`’s `editorialPalette` is split into submodules since #4176, so you can call `background.headline(format)`.